dpkg/pushtopandas.py

```diff
@@ -14,8 +14,9 @@ def push_to_pandas(directory, object_id_cmso):
     storage = dp.push_datapackage(descriptor=descr, backend='pandas')
     print(storage.buckets)
 
-    objects = storage['objects___objectstable']
-    links = storage['links___linkstable']
+    # FIXME: the following is hardwired
```

Comment:
Looks like they have this already scheduled for a change, see https://github.com/frictionlessdata/datapackage-py/issues/110 which they "merged" into https://github.com/frictionlessdata/datapackage-py/issues/132.